cmd/govim/testdata/scenario_workspacemodule/workspace_module.txt

```diff
@@ -0,0 +1,26 @@
+# Verify that gofumpt can be enabled and works
```

Comment:
What is the purpose of this test? Can't really see what it asserts, and also gave it a run with `ExperimentalWorkspaceModule: false` where it still pass.

Reply:
That was sloppy of me, apologies. a) the comment was wrong, and b) I had not correctly asserted on the state of diagnostics. Now fixed on both fronts.